proto/public/api.proto

```diff
@@ -54,6 +54,50 @@ message TapRequest {
   string path = 11;
 }
 
+// A tap request over kubernetes resources.
+message K8sTapRequest {
```

Comment:
The existing `TapRequest` message allows filtering based on source/target ip and port, using these fields:

```
  uint32 toPort = 4;
  string toIP = 5;
  uint32 fromPort = 6;
  string fromIP = 7;
```

Is that functionality still available with the new request message? I couldn't quite figure out how one would filter by ip or port using the new API.

Reply:
This functionality is not currently exposed by the new API, but could be added in the future.

In the short term, I think we should take at _least_ the IP feature out of tap.  The from-port feature also makes very little practical sense to me.  This is in favor of selecting over resource labels.

Reply:
Sounds good to me. We should just make sure that's documented somewhere since it's a change to an existing feature. Maybe open an issue?

Reply:
Ticketed for 0.5.0: #789